hyperstart.go

```diff
@@ -136,11 +121,16 @@ func (h *hyper) buildHyperContainerProcess(cmd Cmd) (*hyperJson.Process, error)
 		envVars = append(envVars, envVar)
 	}
 
+	ioBase, _, err := h.proxy.allocateIO(2)
```

Comment:
So `ioBase` and the `fd` needs to somehow be externally accessible for something like COR to be able to send that back to e.g. the shim.